packages/@vuepress/plugin-darkmode/styles/config.styl

```diff
@@ -0,0 +1,13 @@
+$nightBgcolor ?= #1e1e1e
+$nightFontcolor ?= #9e9e9e
+$nightBordercolor ?= #151310
+$contentClass = '.theme-default-content'
```

Comment:
Configurable?

Reply:
> Configurable?

Yes

Reply:
Well, I was asking why not use `$contentClass ?= ` to avoid override issues

Reply:
> Well, I was asking why not use `$contentClass ?= ` to avoid override issues

The default theme is using = instead of ?=

Reply:
BTW, it's already a pre-defined value in theme default